arch/x86_64/mcount-dynamic.c

```diff
@@ -190,6 +190,7 @@ void mcount_arch_find_module(struct mcount_dynamic_info *mdi,
 	struct arch_dynamic_info *adi;
 	unsigned char fentry_nop_patt1[] = { 0x67, 0x0f, 0x1f, 0x04, 0x00 };
 	unsigned char fentry_nop_patt2[] = { 0x0f, 0x1f, 0x44, 0x00, 0x00 };
+	unsigned char fpatchable_nop_patt[] = { 0x90, 0x90, 0x90, 0x90, 0x90 };
```

Comment:
How about moving them to file level static const variables?

Reply:
It requires another separate change so I made it in https://github.com/namhyung/uftrace/pull/1367/commits/e50b2b459a5d13fe0976cc6188ed1ec17796b751.